### PR TITLE
Kordedekehine patch 3

### PR DESCRIPTION
**Pull request by Kordedekehine**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4388"></a>

### CODE REVIEW

**Review by Krishnapro — CHANGES_REQUESTED** [review]
Required `,` after `}` in line 11. Rest of the looks good

File: Kordedekehine.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    {
+      "name": "check and follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/salami-kehinde-korede-bb85281ab/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Check out my portfolio",
+      "url": "https://salamikorede.netlify.app/",
+      "icon": "FaWeb"
```
Comment (senali-d) [inline Kordedekehine.json:25]: @Kordedekehine This icon is not available in the library. Please double check that.
Here you can find available icons. Please refer: [Available icons](https://linkfree.eddiehub.io/icons)
Icon Suggestion: `FaLink`

Comment (money8203) [reply Kordedekehine.json:25]: > @Kordedekehine This icon is not available in the library. Please double check that. Here you can find available icons. Please refer: [Available icons](https://linkfree.eddiehub.io/icons) Icon Suggestion: `FaLink`

```suggestion
      "icon": "FaLink" 
```

**Review by rahulpandey70 — CHANGES_REQUESTED** [review]
Your json file should be in data folder.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by Panquesito7** [comment]
Looks like it was NOT included in the `data` directory. @SaraJaoude.

**Comment by Kordedekehine** [comment]
> Looks like it was NOT included in the `data` directory. @SaraJaoude.

Can you explain to me how to get it in the data directory

**Comment by rahulpandey70** [comment]
> > Looks like it was NOT included in the `data` directory. @SaraJaoude.
> 
> Can you explain to me how to get it in the data directory

Move your Kordedekehine.json file to data folder and create new PR.

**Comment by SaraJaoude** [comment]
> Looks like it was NOT included in the `data` directory. @SaraJaoude.

Good spot @Panquesito7 but suggest you don't use capitals when feeding back/highlighting a concern in the future as this could upset people.

I have fixed this now @Kordedekehine : https://github.com/EddieHubCommunity/LinkFree/pull/4409

**Pull request merged**